charts/sda-svc/templates/syncapi-secrets.yaml

```diff
@@ -45,7 +45,7 @@ stringData:
       api:
         password: {{ required "sync API password is required" .Values.global.sync.api.password }}
         user: {{ required "sync API user is required" .Values.global.sync.api.user }}
-      brokerQueue: {{ default "mappings" .Values.global.sync.brokerQueue }}
+      brokerQueue: {{ default "mapping_stream" .Values.global.sync.brokerQueue }}
```

Comment:
```suggestion
```

This as well as `centerPrefix` isn't used by the `sync-api`
The entire `remote:` block can be removed as well.